pkg/cli/mirror/mirror.go

```diff
@@ -151,6 +157,19 @@ func (o *MirrorOptions) Complete(cmd *cobra.Command, args []string) error {
 		// If the destination is on disk, made the output dir the
 		// parent dir for the workspace
 		o.Dir = filepath.Join(o.OutputDir, o.Dir)
+	case "oci":
+		if cmd.Flags().Changed("dir") {
+			return fmt.Errorf("--dir cannot be specified with file destination scheme")
```

Comment:
Nit: Is it really "file" destination scheme? Looks like the scheme is "oci", no?

Reply:
right! fixed